### ISSUE TITLE
Deploying stack results in a seemingly unused DynamoDB Table

### ISSUE THREAD

**Issue by ghost** [aws-samples/aws-appsync-iot-core-realtime-dashboard#15]
Greetings,

When I run this stack in my account, I see a DynamoDB Table created with a name like `SensorValue-{generated}-dev`, but when I create some sensors and simulate them, I never see any items added to the Table..

I *can* see the incoming events in the following places though:

* Subscribing to the topic using the web console "MQTT Test Client".
* Querying the configured IoT Analytics dataset.
* Checking relevant Lambda logs in CloudWatch.
* Even in the subscription made from my local React web app..

Could somebody carify what purpose this Table serves here?

**Comment by dave-moser** [comment]
The application logs all of the sensor readings into the DynamoDB table.  When the Lambda function calls the AppSync mutation, the values are logged in the table.  You see no Items in the table?

**Comment by ZaLiTHkA** [comment]
Hi Dave, thanks for getting back to me.

> note: I'm in the process of ditching my secodary GitHub account (`agreeff`), so please excuse the response from different account here.

Interestingly, I did eventually see items in the database, but that came up during an "initial POC demo" with my work colleagues on Tuesday. We were going through a dummy system to investigate a few things, and I was telling them about this DynamoDB Table while I was navigating to the AWS Console page and as I said "for some reason it's always empty", it loaded and suddenly there were thousands of entries (all from my testing). Honestly not sure how or why it happened like this..

Either way, over the past week our own little proof of concept design has now morphed into something different, so I no longer have *this same* architecture any more. As such, I think this issue should probably be closed off for now then.

Considering I'm still in the early days of my AWS adventure, this is very likely due to something that I did incorrectly anyway. 🤦🏼
